### PR TITLE
centre align home img

### PR DESCRIPTION
**Pull request by hereisnaman**
fixes: https://github.com/coding-blocks/motley/issues/130
Added required classes to ProjectX here: https://github.com/coding-blocks/codingblocks.online.projectx/pull/67

Same alignment as shown earlier: 
screen shot 2018-05-15 at 12 24 49 pm

### CODE REVIEW

**Comment by abhishek97** [comment]
Also add dom changes to examples as well

**Comment by hereisnaman** [comment]
@abhishek97 here you go.

File: sass/styles/media-query/media.home.scss [hunk 1]
```diff
@@ -1,36 +1,57 @@
...
-  .home-column{
+  .home-column {
     display: none;
   }
 
-  .home-img{
-    margin: 0;
-    left: 0;
-    width: 100%;
+  .home-img-box {
```
Comment (aayusharora) [inline sass/styles/media-query/media.home.scss]: Why we need an extra class 
.home-img-box

Comment (aayusharora) [reply sass/styles/media-query/media.home.scss]: Can you please provide the URL of that issue @hereisnaman 

We are dealing with content based breakpoints, you need to choose the breakpoint that suits you the most.

Comment (hereisnaman) [reply sass/styles/media-query/media.home.scss]: Here is the link: https://github.com/coding-blocks/codingblocks.online.projectx/pull/36#issuecomment-389043778

Also the search remains centered till 1334px and thats is till when we want the pic to be centered, so either there can be use of more conventional break points or an extra class.

Comment (hereisnaman) [reply sass/styles/media-query/media.home.scss]: @aayusharora updates?

**Comment by abhishek97** [comment]
@aayusharora Do we still need this? please update this.

**Comment by witty123** [comment]
@hereisnaman thanks for the PR
@vdvibhu20 will be working on this so closing it for now

**Pull request closed without merging**